### DISCUSSION TITLE
Looping request using different data params

### DISCUSSION THREAD

**Discussion by Ilie-c** [postmanlabs/postman-flows#282] [Q&A]
Hello, 

I am trying to recreate the loop made in the [30 sept stream](https://youtu.be/j-fkJLbOpjk?t=2966), the one with SWAPI, yet since a lot has changed I cannot find a way now to save the "NEXT" data in a variable or durable then the request to get it... 

Since the method used for save is deprecated now and send request now has variables how will that translate?

**Comment by mitchell-johnson** [comment]
I struggled with this for awhile. what I ended up doing was re-declaring the durable and using the durable as part of the argument. Its not particularly intuitive. 

![Screen Shot 2022-03-31 at 8 16 05 am](https://user-images.githubusercontent.com/845785/160913337-9e8f48ff-27a9-42f5-8242-548886900eb9.png)
